docker/gcp_pudl_etl.sh

```diff
@@ -62,14 +62,22 @@ function copy_outputs_to_distribution_bucket() {
     # Only attempt to update outputs if we have a real value of BUILD_REF
     # This avoids accidentally blowing away the whole bucket if it's not set.
     if [[ -n "$BUILD_REF" ]]; then
-        echo "Removing old $BUILD_REF outputs from GCP distributon bucket." && \
-        gsutil -m -u "$GCP_BILLING_PROJECT" rm -r "gs://pudl.catalyst.coop/$BUILD_REF" && \
+        if [[ "$GITHUB_ACTION_TRIGGER" == "schedule" ]]; then
+            # If running nightly builds, copy outputs to the "nightly" directory
+            # TODO: if doing a tagged release, copy outputs to "stable" as well.
+            DIST_PATH="nightly"
+        else
+            # Otherwise we want to copy them to a directory named after the tag/ref
+            DIST_PATH="$BUILD_REF"
+        fi
```

Comment:
This feels janky. And where/when should we be doing the update to the `stable` outputs?

Reply:
Yeah, it does seem janky, since we're trying to decide where to send the data products based on "is this on a schedule, or not?"

Long-term, we should either pass in "this is nightly/stable/ad-hoc," or "update *this* location in GCS, *this* location in AWS, ..." - flags that actually correspond to our concepts. I'm sort of partial to "nightly/stable/ad-hoc" because that lets us hide the implementations of those within the build script.

But, short-term, this works - though I think we need to add another case to update `stable` outputs when the build ref is `v20xx...`

Reply:
Okay, I think I've got it set up to udpate the `stable` branch on a versioned tag push.